lib/access_import/journal_transform.rb

```diff
@@ -19,6 +19,6 @@ def prepare_attributes(journal, person, assignment, user)
     2 => :conversation,
     3 => :email,
     4 => :feedback,
-    5 => :file
+    5 => :feedback
```

Comment:
Is it intended that way?
```ruby
4 => :feedback,
5 => :feedback
```

Reply:
Yes. They asked this category to be removed. So during the import the entries of this category should be mapped to another one.

Reply:
ah, I see. 👍